src/components/dashjs-player/dashjs-player.css

```diff
@@ -16,6 +16,324 @@ video {
 #myMainVideoPlayer {
   margin-left: auto;
   margin-right: auto;
-  display:block;
+  display: block;
   width: 100%;
-}
\ No newline at end of file
+}
+
+/**************************************************/
```

Comment:
If you like you can move the controlbar to another file, e.g. `controlbar.scss` and add an ` @import "controlbar.scss"` on top (you have to rename this file to `scss` aswell

Reply:
Alright, will do

Reply:
@DanielHabenicht I moved `controlbar.scss` and made ControlBar.js locally. Also added icons but still icons aren't showing for some reason even though the path is right.. would you happen to know why? p.s. I left controlbar styles in `dashjs-player.css ` for now as it gives ```
Unexpected token (Note that you need plugins to import files
           that are not JavaScript)
``` when I import it

Reply:
I will have a look in the afternoon